src/calculateRentalCost.js

```diff
@@ -4,7 +4,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const price = days * 40;
+  let totalCost = price;
+
+  if (days >= 7) {
+    totalCost = price - 50;
+  }
+
+  if (days >= 3 && days < 7) {
+    totalCost = price - 20;
```

Comment:
Similarly, calculate the discounted price directly here: `let totalCost = price - 20;` This will make the code more concise.